freqtrade/pairlist/ConstPairList.py

```diff
@@ -0,0 +1,60 @@
+"""
+Const Pair List provider
+
+Provides pair white list as it configured in config without checking for active markets
+"""
+import logging
+from typing import Any, Dict, List
+
+from freqtrade.exceptions import OperationalException
+from freqtrade.pairlist.IPairList import IPairList
+
+
+logger = logging.getLogger(__name__)
+
+
+class ConstPairList(IPairList):
+
+    def __init__(self, exchange, pairlistmanager,
+                 config: Dict[str, Any], pairlistconfig: Dict[str, Any],
+                 pairlist_pos: int) -> None:
+        super().__init__(exchange, pairlistmanager, config, pairlistconfig, pairlist_pos)
+
+        if self._pairlist_pos != 0:
+            raise OperationalException(f"{self.name} can only be used in the first position "
+                                       "in the list of Pairlist Handlers.")
+
+    @property
+    def needstickers(self) -> bool:
+        """
+        Boolean property defining if tickers are necessary.
+        If no Pairlist requires tickers, an empty List is passed
+        as tickers argument to filter_pairlist
+        """
+        return False
+
+    def short_desc(self) -> str:
+        """
+        Short whitelist method description - used for startup-messages
+        -> Please overwrite in subclasses
+        """
+        return f"{self.name}"
+
+    def gen_pairlist(self, cached_pairlist: List[str], tickers: Dict) -> List[str]:
+        """
+        Generate the pairlist
+        :param cached_pairlist: Previously generated pairlist (cached)
+        :param tickers: Tickers (from exchange.get_tickers()).
+        :return: List of pairs
+        """
+        return self._config['exchange']['pair_whitelist']
```

Comment:
if i'm not mistaking, this is almost an exact copy of StaticPairList with just this line changed.

In that case, I'd rather have an "allowInactive" (or "validatePairs" or similar) flag on the "StaticPairList" configuration (look at `VolumePairlist` or `AgeFilter` on how that might be done) that does (or does not) do the validation. 
A lot less duplicate code, while archiving the same thing.

Reply:
That's a nice mechanism, I remove the new handler and added allow_inactive to StaticPairList.